crates/manifest/src/resolved.rs

```diff
@@ -0,0 +1,219 @@
+use std::{collections::BTreeMap, path::PathBuf};
+
+use serde::{Deserialize, Serialize};
+use serde_value::Value;
+use serde_with::{hex::Hex, As};
+
+use crate::ManifestVersion;
+
+/// ResolvedManifest is a "resolved" application manifest, ready to be used by an executor.
```

Comment:
@itowlson

Reply:
Not sure what I'm being pinged on here but this looks like it is used both as the internal ready-to-use format _and_ a serialisation format, and I have never had good experiences of mixing those two... was that what you were flagging me for?

Reply:
Just flagging as this is my draft of a "spin.lock" which we have discussed. _Not_ reusing serialization structures for application logic is one of my major goals with all this refactoring noise.